runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -12,23 +12,25 @@ LanguageTree.__index = LanguageTree
 -- @param source Can be a bufnr or a string of text to parse
 -- @param lang The language this tree represents
 -- @param opts Options table
--- @param opts.queries A table of language to injection query strings.
---                     This is useful for overriding the built-in runtime file
---                     searching for the injection language query per language.
+-- @param opts.injections A table of language to injection query strings.
+--                        This is useful for overriding the built-in runtime file
+--                        searching for the injection language query per language.
 function LanguageTree.new(source, lang, opts)
   language.require_language(lang)
   opts = opts or {}
 
-  local custom_queries = opts.queries or {}
+  assert(not opts.queries, "'queries' is no longer supported. Use 'injections' now")
```

Comment:
Maybe using an assert here is a bit to harsh ?
Just saying that to the user, and falling back would be better IMO.

Reply:
So `print`?

Reply:
done